src/client/app/components/LineChartComponent.tsx

```diff
@@ -73,14 +73,13 @@ export default function LineChartComponent() {
 		return <SpinnerComponent loading height={50} width={50} />;
 	}
 
-
-	// Check if there is at least one valid graph
-	const enoughData = data.find(data => data.x!.length > 1);
 	// Customize the layout of the plot
 	// See https://community.plotly.com/t/replacing-an-empty-graph-with-a-message/31497 for showing text not plot.
 	if (data.length === 0) {
 		return <h1>{`${translate('select.meter.group')}`}	</h1>;
-	} else if (!enoughData) {
+	}
+	// Checks if there is enough data for at least one valid graph
+	else if (!data[0].x) {
```

Comment:
This change seems to have stopped the message about no data in range for the meter. I found this by changing the range outside all points.